frontend/src/Components/MonitorToggleButton.js

```diff
@@ -3,18 +3,19 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import SpinnerIconButton from 'Components/Link/SpinnerIconButton';
 import { icons } from 'Helpers/Props';
+import translate from 'Utilities/String/translate';
 import styles from './MonitorToggleButton.css';
 
 function getTooltip(monitored, isDisabled) {
   if (isDisabled) {
-    return 'Cannot toggle monitored state when series is unmonitored';
+    return translate('CannotToggleMonitoredWhenSeriesIsUnmonitored');
```

Comment:
`ToggleMonitoredSeriesUnmonitored` this would keep it with the other related translations and shortens it up.